### ISSUE TITLE
18448513: Lack of a system-wide dark mode is an accessibility issue.

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#3076]
#### Description

Currently, there is no system-wide feature to allow a dark mode. A universal low-contrast mode would be useful at nighttime, and for people who with certain vision issues (http://www.cgpgrey.com/blog/dark-mode-as-accessibility-feature), as a high contrast white background can incredibly frustrating to use.

The inverted colors is not a good solution — colors in photographs and UI lose meaning.

In addition, inverting colors is not easily accessible, even when triple tapping a home button, which requires the user to dig through the settings app to activate.

Ideally, there would be a system-wide switch that turns on dark mode permanently (like a more global Yosemite-style dark mode), or in situations when the screen brightness drops below a certain level.
## 

Product Version: iOS 8
Created: 2014-09-24T23:58:47.487947
Originated: 2014-09-24T00:00:00
Open Radar Link: http://www.openradar.me/18448513
